tools/ci_build/github/azure-pipelines/nuget/templates/cpu-nocontribops-arm64.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     NuPackScript: |
      msbuild $(Build.SourcesDirectory)\csharp\OnnxRuntime.CSharp.proj /p:Configuration=RelWithDebInfo /t:CreatePackage
      copy $(Build.SourcesDirectory)\csharp\src\Microsoft.ML.OnnxRuntime\bin\RelWithDebInfo\*.nupkg $(Build.ArtifactStagingDirectory)
```

Comment:
Use $(Build.BinariesDirectory) instead of sources --  make it consistent with below.